doc/index.rst

```diff
@@ -119,12 +119,15 @@ Software Foundation Code of Conduct <coc_>`_ in everything we do.
 
 Check the :doc:`FAQ <faq/index>` and the :doc:`API <api/index>` docs.
 
-For help, join the `gitter channel`_ and the matplotlib-users_,
-matplotlib-devel_, and matplotlib-announce_ mailing lists, or check out the
-Matplotlib tag on stackoverflow_.  The `search <search.html>`_ tool searches
-all of the documentation, including full text search of over 350 complete
+For help, write us at `discourse.matplotlib.org`_.  If its something quick
```

Comment:
The discourse is for more than just help though? Maybe something like " Join our community at 'discourse' to get help, discuss contributing & development, and share your work?

Reply:
Did this plus a bit of re-formatting to bulleted list